.github/ISSUE_TEMPLATE/ticket_template1.md

```diff
@@ -0,0 +1,27 @@
+🚨Please review the [guidelines for feature ticket](../HOWTOWRITEATICKETS.md) to this repository if unsure about what to write..
+
++## Problem / Motivation
+Describe your issue here. You can you the user story format.
+- As a ... Who?
+- I would like ... what?
+- So that ... why?
+
+
++## Proposed solution and tasks
+We could build this.... and it will solve that...
+
+List of tasks for this ticket to be complete:
+- [ ] A descriptive of a task
```

Comment:
Tasks can probably link directly to other issues with an understandable title, but I understand at ticket creation time these do not exist yet.